src/h5web/visualizations/shared/models.ts

```diff
@@ -1,3 +1,8 @@
 export type Domain = [number, number];
 
-export type AxisOffsets = [number, number];
+export type AxisOffsets = { left: number; bottom: number };
```

Comment:
I changed AxisOffsets signature to be consistent with AxisDomains. It also removes the need to destructure the array several times.

It might be worth merging AxisOffsets and AxisDomains once we agree on the axis configuration options.

Reply:
Good call